pinot-common/src/main/java/com/linkedin/pinot/common/config/IndexingConfig.java

```diff
@@ -72,6 +72,9 @@
   @ConfigKey("starTreeIndexSpec")
   private StarTreeIndexSpec _starTreeIndexSpec;
 
+  @ConfigKey("starTreeIndexConfigs")
```

Comment:
whats the difference between startreeindexSpec and starTreeIndexConfigs. Do we need both?

Reply:
starTreeIndexSpec is for old star-tree, while starTreeIndexConfig is for new star-tree. Once we fully migrate to the new tree, we can remove the starTreeIndexSpec.